docs/ref/models/querysets.txt

```diff
@@ -1354,6 +1354,30 @@ one, doing so will result in an error.
     reader, is slightly faster and consumes a little less memory in the Python
     process.
 
+    For example::
+
+         # Both these models use the same underlying database table
+         # However, you would use an unmanaged model with fewer fields if they
+         # were the fields most often used.
+         class CommonlyUsedModal(models.Model):
+             f1 = models.CharField(max_length=10)
+             f2 = models.CharField(max_length=10)
+
+             class Meta:
+                 managed = False
+                 db_table = 'app_largetable'
+
+         class ManagedModel(models.Model):
+             f1 = models.CharField(max_length=10)
+             f2 = models.CharField(max_length=10)
+             f3 = models.CharField(max_length=10)
+
+             class Meta:
+                 db_table = 'app_largetable'
+
+         # Same as `ManagedModel.objects.all().defer("f3")` but more succinct
+         CommonlyUsedModal.objects.all()
+
```

Comment:
I considered that; and actually think using a proxy is probably more idiomatic, but the context of this change is specifically about using an unmanaged model, and the final sentence of the note has a point: using this approach doesn't create a modal with a bunch of deferred fields; is more explicit as to what is being loaded; and is probably slightly more optimized.